components/radio/Radio.jsx

```diff
@@ -48,38 +51,37 @@ export default {
   },
 
   render() {
-    const { $slots, radioGroupContext: radioGroup } = this;
+    const { $slots, radioGroupContext: radioGroup, $attrs } = this;
     const props = getOptionProps(this);
```

Comment:
const children = $slots.default;
to
const children = $slots.default && $slots.default();